src/PresentationalComponents/PathwaysPanel/PathwaysPanel.js

```diff
@@ -17,14 +17,18 @@ import messages from '../../Messages';
 import propTypes from 'prop-types';
 import { useGetPathwaysQuery } from '../../Services/Pathways';
 import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
 
 const PathwaysPanel = () => {
   const intl = useIntl();
+  const filters = useSelector(({ filters }) => filters.pathState);
   const [expanded, setExpanded] = useState(
     JSON.parse(localStorage.getItem('advisor_pathwayspanel_expanded') || 'true')
   );
   const { data, isLoading, isFetching, isError } = useGetPathwaysQuery({
+    sort: filters.sort,
```

Comment:
You could just spread the `filters` const in, or?

Reply:
We dont want all the same params passed in. Mostly just want how they're sorted to be the same. I had throw in offset just incase we changed it in the future. The limit for them need to be different, but I guess I can manipulate it to only show the first 3 in the array (limit is normally 20) if that seems nicer to just spread in {...filters}

Reply:
maybe rewriting line 24, to:

```js
const { sort, offset } = useSelector(({ filters: { pathState } }) => pathState);
```

then you can write `{ sort, offset, limit: 3 }` here.

Reply:
hmmm, that does look nicer, ill shoot that up